x-pack/plugins/triggers_actions_ui/server/data/lib/time_series_query.ts

```diff
@@ -82,13 +116,25 @@ export async function timeSeriesQuery(
       aggParent.aggs.groupAgg.terms.order = {
         sortValueAgg: sortOrder,
       };
+    } else if (includeConditionInQuery) {
+      aggParent.aggs.groupAgg.aggs = {
+        conditionSelector: {
+          bucket_selector: {
+            buckets_path: {
+              [TIME_SERIES_BUCKET_SELECTOR_PATH_NAME]: '_count',
+            },
+            script: conditionParams.conditionScript,
```

Comment:
I was a little worried we had to use scripting for this, but seems fine since it's over the existing aggs, so subject to the limit anyway.  Seems fine.